doc/kubernetes/container-image-management.md

```diff
@@ -0,0 +1,76 @@
+# Why container image management
+
+After we support container upgrade via k8s, for the container image, we have two problem to be fixed.
+- After we upgrade one container to a higher version via k8s, if k8s control plane is down and worker reboot, container need to go back to local mode, local mode container should run the higher version other than the original version.
+- Upgrade will bring a new version image to device, after upgrade many times, the disk will be full, need to avoid this.
+
+So we need to manage the container image to fix these two problems
+
+# Existing designs need to know first
```

Comment:
don't need a dedicated section for this, we can just have a very short and concise introduction for the local/kube mode.